### ISSUE TITLE
review and update background image description.

### ISSUE THREAD

**Issue by StephDriver** [BirkbeckCTP/hourglass#361]
from #324 

While these are decorative, they also set the scene for each page.  To ignore them for assistive tech users would remove a key part of our branding and design from their experience.  They currently have some alt-text, which I agree is required, but the content of this needs to be revised for clarity.  For example when landing on the home page this is how that page begins when being read aloud (using Voiceover on iOS):

```
Open Library of Humanities.  
Toggle Navigation.
We are the Open Library of Humanities.
Woman with hoop earrings dancing and waving arms.
Woman burying her face in the pages of a book.
For free.
For everyone.
Forever.
```
The two lines about women are the descriptions of the two images as the top of the page. These have distinct visual styling that links across the whole site - and yet here none of that comes through. I suggest we need a form of text that identifies images as branding/background that fits with the theme so that these images can be given their own alt-text styling, that also fits with the branding.  This might be something to think about more widely and add to the asset library alongside the images themselves.

On mobile, some of these background images are part off the side of the screen - so the description then doesn't match with what is visible on the page because it id describing the whole image.  This needs a rethink, as image description should match the visible image.

Similarly other background images are described where they fall, without necessarily fitting within the flow of the text of the page.  But not all background images are described (for example, the image at the top of the OA glossary page is not described).
1. reconsider how background images should be described in line with the visual branding
2. reconsider the description in regards to when the background image is read out in the context of surrounding paragraphs.
3. consider how first background image at the top of the page sets the scene/branding identity and how an equivalent may be done with the description that makes sense in the context of hearing that page.
